nipype/interfaces/base.py

```diff
@@ -28,32 +26,24 @@
 import select
 import subprocess
 import sys
-import random
 import time
-import fnmatch
 from textwrap import wrap
 from datetime import datetime as dt
 from dateutil.parser import parse as parseutc
 from warnings import warn
 
 
-from .traits_extension import (traits, Undefined, TraitDictObject,
-                               TraitListObject, TraitError,
-                               isdefined, File, Directory,
-                               has_metadata)
-from ..utils.filemanip import (md5, hash_infile, FileNotFoundError,
-                               hash_timestamp, save_json,
-                               split_filename)
-from ..utils.misc import is_container, trim, str2bool
-from ..utils.provenance import write_provenance
-from .. import config, logging, LooseVersion
-from .. import __version__
-from ..external.six import string_types, text_type
+from nipype import config, logging, LooseVersion, __version__
```

Comment:
why did these become absolute imports? py3 recommends relative imports for packages.